docs/01_about.md

```diff
@@ -9,7 +9,7 @@ scientists in the field of atmospheric science. The purpose is to have a
 tool that simplifies the process for planning a scientific flight in
 which parameters of the atmosphere are measured. The research aircrafts
 typically carry a comprehensive scientific payload comprised of data
-aquisition instruments by different companies and research institutions.
+acquisition instruments by different companies and research institutions.
```

Comment:
deepl.com knows both words.

(which word is better, @joernu76 ?)

Reply:
With "c" is correct. Without "c" is a common mis-spelling and deepl is trained by the internet, isn't it? Not the best reference for spelling I'd say.